packages/graph/hash_graph/lib/graph/src/knowledge/link/mod.rs

```diff
@@ -64,29 +64,52 @@ impl Link {
 /// metadata.
```

Comment:
I think this doc is incorrect now

Reply:
Thanks for spotting, addressed in https://github.com/hashintel/hash/pull/1199/commits/f342d3263425fd088678f95117fe445dac731c69